src/middlewares/phoenix/actions/index.js

```diff
@@ -201,8 +201,11 @@ export function connectToPhoenixChannelForEvents({
             channel,
           })
         );
-        dispatch(endPhoenixChannelProgress({ channelTopic, loadingStatusKey: channelTopic }));
+        dispatch(
+          endPhoenixChannelProgress({ dispatch, channelTopic, loadingStatusKey: channelTopic })
+        );
         if (channelResponseEvent) {
+          console.log(channelResponseEvent, dispatch, response);
```

Comment:
You can modify local copy to add additional logging, it isn't require pushing to dependency repo